plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineSdkClasspathContainer.java

```diff
@@ -8,17 +8,14 @@
 import org.eclipse.jdt.core.IClasspathEntry;
 import org.eclipse.jdt.core.JavaCore;
 
+import com.google.cloud.tools.appengine.cloudsdk.CloudSdk;
 import com.google.cloud.tools.eclipse.sdk.CloudSdkProvider;
 
 public final class AppEngineSdkClasspathContainer implements IClasspathContainer {
 
-  // TODO should be changed once app-tools-lib can provide the directory
-  // https://github.com/GoogleCloudPlatform/app-tools-lib-for-java/issues/134
-  private static final String SDK_JAR = "platform/google_appengine/google/appengine/tools/java/lib/appengine-tools-api.jar";
+  private static final CloudSdk CLOUD_SDK = new CloudSdkProvider().getCloudSdk();
```

Comment:
Removed static.

I suspect you want to pass in a CloudSdk instance so we always get the freshest Cloud SDK we have. In that case, could it make sense to just call CloudSdkProvider every time we want it? Or are you worried this will incur in unnecessary FS checks?